ocaml/libs/tracing/tracing.ml

```diff
@@ -138,19 +163,20 @@ module Span = struct
             )
         in
         let status_code = Status.Error in
-        let exn_tags =
+        let exn_attributes =
           [
             ("exception.message", msg)
           ; ("exception.stacktrace", stacktrace)
           ; ("exception.type", exn_type)
+          ; ("error", "true")
           ]
         in
         match span.status.status_code with
         | Unset ->
             {
               span with
               status= {status_code; description}
-            ; tags= span.tags @ exn_tags
+            ; attributes= span.attributes @ exn_attributes
```

Comment:
Could we end up with duplicate keys? I assume these lists are short.

Reply:
attributes also use namespaces. Default attributes begin with `xs.`, exn with `exception.` and user with `user.`. There is one which does nto have a namespace `error` and this is to work with Jaeger's internal system to detect if a span had an error.